go.mod

```diff
@@ -4,6 +4,7 @@ require (
 	github.com/go-openapi/jsonpointer v0.19.3
 	github.com/go-openapi/jsonreference v0.19.2
 	github.com/go-openapi/swag v0.19.5
+	github.com/kevinburke/go-bindata v3.13.0+incompatible // indirect
```

Comment:
oh you did the go get -u ... from the local repo... But this is not needed. Please remove this from the dependencies

Reply:
Removed.